examples/with-claims/api/claims/RolesClaim.ts

```diff
@@ -0,0 +1,91 @@
+import Session, { SessionClaim, SessionClaimValidator, SessionContainer } from "supertokens-node/recipe/session";
+import { roleDB } from "../mockRoleDb";
+
+class RolesClaimClass extends SessionClaim<string[]> {
+    constructor() {
+        super("st-r");
+        this.hasRole = {
+            validatorTypeId: this.key,
+            including: (role: string) => ({
```

Comment:
I don't think we should `including`. Just `RolesClaimClass.hasRole([".."]);`

Reply:
This is for symmetry with the front-end, where we may want to use `hasRole.validatorTypeId`

Reply:
`RoleClaim.hasRoles.includes("admin")`
`RoleClaim.hasRoles.includesAll(["admin", "..."])`
`RoleClaim.hasRoles.includesAny(["admin", "..."])`
`RoleClaim.getRolesFromPayload(accessTokenPayload)`

(Also, these should go in supertokens-web-js?)